test/e2e/set/scyllacluster/scyllacluster_replace.go

```diff
@@ -106,9 +145,23 @@ var _ = g.Describe("ScyllaCluster replace", func() {
 		oldHosts := hosts
 		hosts = getScyllaHostsAndWaitForFullQuorum(ctx, f.KubeClient().CoreV1(), sc)
 		o.Expect(hosts).To(o.HaveLen(len(oldHosts)))
-		o.Expect(hosts).NotTo(o.ConsistOf(oldHosts))
 		err = di.SetClientEndpoints(hosts)
 		o.Expect(err).NotTo(o.HaveOccurred())
 		verifyCQLData(ctx, di)
-	})
+
+		framework.By("Verifying ScyllaDB config")
+
+		configClient, err := utils.GetScyllaConfigClient(ctx, f.KubeClient().CoreV1(), sc, replacedNodeService.Spec.ClusterIP)
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		err = validateScyllaConfig(ctx, configClient, preReplaceService)
+		o.Expect(err).NotTo(o.HaveOccurred())
+	},
+		g.Entry("using node clusterIP address when version of Scylla OS is <5.2", scyllaOSImageRepository, "5.1.4", validateReplaceViaClusterIPAddress),
+		g.Entry("using node clusterIP address when version of Scylla Enterprise is <2023.1", scyllaEnterpriseImageRepository, "2022.2.11", validateReplaceViaClusterIPAddress),
+		g.Entry("using node Host ID when version of Scylla OS is >=5.2", scyllaOSImageRepository, "5.2.0", validateReplaceViaHostID),
+
+		// TODO: Enable test when ScyllaDB Enterprise 2023.1 is released
```

Comment:
can we use `2023.1.0-rc7` for this specific test? otherwise it needs an issue link

Reply:
No, `rc < final version`, so Operator executes address based procedure. 2023.1 should be out next week, lets see before merge whether we need to add an issue or enable the test.

Reply:
in that case - can you add a link to a tracking issue to enable it when it becomes GA?

Reply:
added